docker/base/create_cluster.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+mkdir -p /nodes
+echo -n > /nodes/nodemap
+for PORT in $(seq 16379 16384); do
+  mkdir -p /nodes/$PORT
+  if [[ -e /redis.conf ]]; then
+    cp /redis.conf /nodes/$PORT/redis.conf
+  else
+    touch /nodes/$PORT/redis.conf
+  fi
+  cat << EOF >> /nodes/$PORT/redis.conf
+port $PORT
+daemonize yes
+logfile /redis.log
+dir /nodes/$PORT
+EOF
+  redis-server /nodes/$PORT/redis.conf
```

Comment:
Suggestion: Validate that the redis-server comes up properly:

```
if [ $? -ne 0 ]; then
    echo "Redis failed to start, exiting."
    exit 3
fi

Reply:
Perhaps we should also run redis with nohup?

Reply:
we're starting redis with --daemonize, do you think we should also add nohup?